Index: app/src/main/java/com/ss_team_1/koibitoshuuchuu/presentation/MainActivity.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.ss_team_1.koibitoshuuchuu.presentation\r\n\r\nimport androidx.activity.ComponentActivity\r\n\r\nclass MainActivity: ComponentActivity() {\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/ss_team_1/koibitoshuuchuu/presentation/MainActivity.kt b/app/src/main/java/com/ss_team_1/koibitoshuuchuu/presentation/MainActivity.kt
--- a/app/src/main/java/com/ss_team_1/koibitoshuuchuu/presentation/MainActivity.kt	(revision 89078e96ffa50c6929056216354f6605c20e6543)
+++ b/app/src/main/java/com/ss_team_1/koibitoshuuchuu/presentation/MainActivity.kt	(date 1654017028705)
@@ -1,6 +1,82 @@
 package com.ss_team_1.koibitoshuuchuu.presentation
 
+import android.os.Bundle
 import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.Canvas
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Paint
+import androidx.compose.ui.graphics.drawscope.drawIntoCanvas
+import androidx.compose.ui.graphics.nativeCanvas
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.ss_team_1.koibitoshuuchuu.ui.theme.KoiBitoShuuChuuTheme
+import com.ss_team_1.koibitoshuuchuu.ui.theme.mamelonFamily
 
 class MainActivity: ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
+            KoiBitoShuuChuuTheme {
+                LoverFocusApp()
+            }
+        }
+    }
+}
+
+@Composable
+fun LoverFocusApp(){
+    Surface(color = MaterialTheme.colors.primary) {
+        Text(text = "Welcome，你好", modifier = Modifier.padding(24.dp),fontFamily = mamelonFamily, fontWeight = FontWeight.Normal)
+        Canvas(
+            modifier = Modifier.fillMaxSize(),
+            onDraw = {
+                drawIntoCanvas {
+                    it.nativeCanvas.drawText(
+                        "Sample",
+                        0f,
+                        120.dp.toPx(),
+                        textPaintStroke
+                    )
+                    it.nativeCanvas.drawText(
+                        "Sample",
+                        0f,
+                        120.dp.toPx(),
+                        textPaint
+                    )
+                }
+            }
+        )
+
+    }
+}
+val textPaintStroke = Paint().asFrameworkPaint().apply {
+    isAntiAlias = true
+    style = android.graphics.Paint.Style.STROKE
+    textSize = 64f
+    color = android.graphics.Color.BLACK
+    strokeWidth = 12f
+    strokeMiter= 10f
+    strokeJoin = android.graphics.Paint.Join.ROUND
+}
+
+val textPaint = Paint().asFrameworkPaint().apply {
+    isAntiAlias = true
+    style = android.graphics.Paint.Style.FILL
+    textSize = 64f
+    color = android.graphics.Color.WHITE
+}
+@Preview
+@Composable
+fun DefaultPreview() {
+    KoiBitoShuuChuuTheme {
+        LoverFocusApp()
+    }
 }
\ No newline at end of file
Index: app/src/main/java/com/ss_team_1/koibitoshuuchuu/ui/theme/color.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.ss_team_1.koibitoshuuchuu.ui.theme\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/ss_team_1/koibitoshuuchuu/ui/theme/color.kt b/app/src/main/java/com/ss_team_1/koibitoshuuchuu/ui/theme/color.kt
--- a/app/src/main/java/com/ss_team_1/koibitoshuuchuu/ui/theme/color.kt	(revision 89078e96ffa50c6929056216354f6605c20e6543)
+++ b/app/src/main/java/com/ss_team_1/koibitoshuuchuu/ui/theme/color.kt	(date 1654015663997)
@@ -1,2 +1,16 @@
 package com.ss_team_1.koibitoshuuchuu.ui.theme
 
+import androidx.compose.ui.graphics.Color
+
+val Pink = Color(0xfff8bbd0)
+val GreenBlue = Color(0xff94e7e1)
+val DarkGreenBlue = Color(0xff09756D)
+val Purple200 = Color(0xFFBB86FC)
+val Purple500 = Color(0xFF6200EE)
+val Purple700 = Color(0xFF3700B3)
+val Teal200 = Color(0xFF03DAC5)
+val Navy = Color(0xFF073042)
+val Blue = Color(0xFF4285F4)
+val LightBlue = Color(0xFFD7EFFE)
+val Chartreuse = Color(0xFFEFF7CF)
+
Index: app/src/main/java/com/ss_team_1/koibitoshuuchuu/ui/theme/Theme.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/ss_team_1/koibitoshuuchuu/ui/theme/Theme.kt b/app/src/main/java/com/ss_team_1/koibitoshuuchuu/ui/theme/Theme.kt
new file mode 100644
--- /dev/null	(date 1654011059877)
+++ b/app/src/main/java/com/ss_team_1/koibitoshuuchuu/ui/theme/Theme.kt	(date 1654011059877)
@@ -0,0 +1,55 @@
+package com.ss_team_1.koibitoshuuchuu.ui.theme
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.darkColors
+import androidx.compose.material.lightColors
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+
+@Composable
+fun KoiBitoShuuChuuTheme(
+    darkTheme: Boolean = isSystemInDarkTheme(),
+    content: @Composable () -> Unit
+) {
+    val colors = /*if (darkTheme) {
+        DarkColorPalette
+    } else {*/
+        LightColorPalette
+    //}
+
+    MaterialTheme(
+        colors = colors,
+        typography = Typography,
+        shapes = Shapes,
+        content = content
+    )
+}
+
+/*private val DarkColorPalette = darkColors(
+    surface = Blue,
+    onSurface = Navy,
+    primary = Navy,
+    onPrimary = Chartreuse
+)*/
+
+
+private val LightColorPalette = lightColors(
+    surface = GreenBlue, //再背景上的按鈕
+    onSurface = DarkGreenBlue,
+    primary = Pink,
+    onPrimary = Color.White,
+    secondary = GreenBlue,
+    onSecondary = DarkGreenBlue
+
+
+
+    /* Other default colors to override
+    background = Color.White,
+    surface = Color.White,
+    onPrimary = Color.White,
+    onSecondary = Color.Black,
+    onBackground = Color.Black,
+    onSurface = Color.Black,
+    */
+)
Index: app/src/main/java/com/ss_team_1/koibitoshuuchuu/ui/theme/Type.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/ss_team_1/koibitoshuuchuu/ui/theme/Type.kt b/app/src/main/java/com/ss_team_1/koibitoshuuchuu/ui/theme/Type.kt
new file mode 100644
--- /dev/null	(date 1654016287895)
+++ b/app/src/main/java/com/ss_team_1/koibitoshuuchuu/ui/theme/Type.kt	(date 1654016287895)
@@ -0,0 +1,34 @@
+package com.ss_team_1.koibitoshuuchuu.ui.theme
+
+import androidx.compose.material.Typography
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.sp
+import com.ss_team_1.koibitoshuuchuu.R
+
+// Set of Material typography styles to start with
+val Typography = Typography(
+    body1 = TextStyle(
+        fontFamily = FontFamily.Default,
+        fontWeight = FontWeight.Normal,
+        fontSize = 16.sp
+    )
+
+    /* Other default text styles to override
+    button = TextStyle(
+        fontFamily = FontFamily.Default,
+        fontWeight = FontWeight.W500,
+        fontSize = 14.sp
+    ),
+    caption = TextStyle(
+        fontFamily = FontFamily.Default,
+        fontWeight = FontWeight.Normal,
+        fontSize = 12.sp
+    )
+    */
+)
+val mamelonFamily = FontFamily(
+    Font(R.font.mamelon, FontWeight.Normal)
+)
Index: app/src/main/java/com/ss_team_1/koibitoshuuchuu/ui/theme/Shape.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/ss_team_1/koibitoshuuchuu/ui/theme/Shape.kt b/app/src/main/java/com/ss_team_1/koibitoshuuchuu/ui/theme/Shape.kt
new file mode 100644
--- /dev/null	(date 1654009147609)
+++ b/app/src/main/java/com/ss_team_1/koibitoshuuchuu/ui/theme/Shape.kt	(date 1654009147609)
@@ -0,0 +1,11 @@
+package com.ss_team_1.koibitoshuuchuu.ui.theme
+
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Shapes
+import androidx.compose.ui.unit.dp
+
+val Shapes = Shapes(
+    small = RoundedCornerShape(4.dp),
+    medium = RoundedCornerShape(4.dp),
+    large = RoundedCornerShape(0.dp)
+)
\ No newline at end of file
